app/controllers/weets.js

```diff
@@ -0,0 +1,18 @@
+const geekJokesService = require('../services/geek_jokes');
+const errors = require('../errors');
+
+const getWeet = async (req, res, next) => {
+  try {
+    const { data } = await geekJokesService.getGeekJoke();
+    return res.status(200).send({
+      data,
+      message: 'Weet was obtained'
```

Comment:
this string "Weet was obtained" should be in a constants file, so that later in the tests, you only import the constant and you don't have to rewrite the message

Reply:
ok